src/components/Gradebook/index.jsx

```diff
@@ -346,6 +377,7 @@ export default class Gradebook extends React.Component {
                   }
                   value={this.state.filterValue}
                 />
+                <small className="form-text text-muted search-help-text">Search by username, email, or student key</small>
```

Comment:
Using the same element/classname combination as the help text in the language section of the new account settings page.

Reply:
TODO: If email is not being displayed in the gradebook (e.g. for a non-Master's course), we may want to consider removing email from these instructions to avoid confusing the user.